src/pages/home/report/ReportActionCompose/ComposerWithSuggestions.js

```diff
@@ -445,6 +448,28 @@ function ComposerWithSuggestions({
         textInputRef.current.blur();
     }, []);
 
+    const focusWithScrolledToBottom = useCallback(() => {
+        // Using `shouldAutoFocus` check to determine whether the component should be focused or not.
+        if (!textInputRef.current || !willBlurTextInputOnTapOutside || !shouldAutoFocus) {
+            return;
+        }
+
+        InteractionManager.runAfterInteractions(() => {
+            updateMultilineInputRange(textInputRef.current);
+            focus(); // This will focus the composer after its fully rendered.
```

Comment:
Unsure why we need to force focus here?

Reply:
https://github.com/Expensify/App/blob/f070f8b5dc8d78b9593cd239d4fd2956c1ec470e/src/pages/home/report/ReportActionCompose/ComposerWithSuggestions.js#L535

If you check above line of code we are disabling autoFocus as we wan't to focus manually after scrollHeight and selection are updated.

Reply:
So without that, the selection wouldn't be set to the end, is that what you mean? Or what side-effect is there if we _don't_ add that?

Reply:
https://github.com/Expensify/App/assets/78416198/04e533d7-2495-4623-b7a0-27109b8206c4

Without that Composer won't get Focused. As you can see the Cursor isn't there in the end.

Reply:
Sorry this still isn't really making sense to me. If I remove your change to `autoFocus` and just have a simple `useEffect` that runs `updateMultilineInputRange`, it works fine (on desktop at least). I'm struggling to understand the reason why all this extra logic is necessary here.

Reply:
Did you checked my video @jjcoffee ?

Reply:
Lets have a quick chat in Slack here its harder to chat.

Reply:
@jeet-dhandha Yes, is that with `autoFocus` set back to `shouldAutoFocus`? I don't get the same behaviour if so...

Reply:
The original issue also doesn't show any problem with focusing on the composer. That's why I'm quite sceptical about adding extra focus calls here!